config/keys.go

```diff
@@ -65,5 +71,27 @@ func LoadSigningKeys() (*SigningKeys, error) {
 		}
 		return nil, err
 	}
+
+	if err := validateKeys(&config); err != nil {
+		return nil, err
+	}
+
 	return &config, nil
 }
+
+func validateKeys(config *SigningKeys) error {
+	keys := config.Keys
+	uniqMap := make(map[string]struct{}, len(keys))
+	for _, key := range keys {
+		if _, ok := uniqMap[key.Name]; ok {
```

Comment:
Do we need to check an empty string key name?

Reply:
We don't have that in specification but it's a good point, should we restrict it to alpha numeric?